Assets/Source/Components/Player/PlayerBehavior.cs

```diff
@@ -56,9 +56,22 @@ private void UpdatePlayerControls()
             //This version just teleports the shit a little to the left/right
             //Final version should include actual movement
 
-            if (InputManager.IsKeyPressed(InputConstants.K_SHOOT))
+            float dashDistance = .5f; //TODO: make this a constant somewhere
```

Comment:
should this be a constant? What if we want it to be "upgradable" so that their dash distance is farther?